src/tests/test_kube_api.py

```diff
@@ -2,13 +2,15 @@
 import json
 import os
 import tempfile
+from contextlib import suppress
 from typing import Callable, List, Optional
 
 import openshift as oc
 import pytest
 import waiting
+from _pytest.fixtures import FixtureLookupError, FixtureRequest
```

Comment:
```suggestion
from pytest.fixtures import FixtureLookupError, FixtureRequest
```

Reply:
It's not available for import from `pytest`